.circleci/config.yml

```diff
@@ -115,9 +115,6 @@ workflows:
           matrix:
             parameters:
               goversion:
-                - "1.14"
-                - "1.15"
-                - "1.16"
```

Comment:
I'm thinking we could drop _all_ versions but the one the layer is built against. The layer is a binary distribution, not a library.

Reply:
To be dealt with separately in #100.